forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -215,6 +219,8 @@ else if (c.hasKeyword(Keyword.WITHER)) {
             } else {
                 value -= subValue(50, "doesnt-untap");
             }
+        } else {
+            value -= subValue(10 * c.getCounters(CounterEnumType.SHIELD), "stunned");
```

Comment:
@tool4ever didn't you mean Stun counter?

Reply:
for sure, thanks